data/mods/Aftershock/items/gun/10mm.json

```
@@ -0,0 +1,36 @@
+[
+  {
+    "id": "afs_orion_84K",
+    "copy-from": "pistol_base",
+    "looks_like": "glock_17",
+    "type": "GUN",
+    "name": { "str": "Orion 84K" },
+    "description": "Commonly carried by commercial spacers who can afford it, the Orion 84K manufactured by Orion Armistice doesn't particulary shine or show anything special but it keeps you safe on long journeys in the vast dark of Orion's Arm.",
```

```suggestion
    "description": "Commonly carried by shipping crews that can afford it, Orion Armistice's 84K pistol doesn't particulary shine or show anything special but it keeps you safe on long journeys in the vast dark of Orion's Arm.",
```
Edited to describe it as a pistol, and to remove some repetitions of "Orion".
Aditionally, I would consider a different manufacturer to avoid repeating Orion at all.